python/dask_cudf/dask_cudf/backends.py

```diff
@@ -64,27 +60,34 @@ def _nonempty_series(s, idx=None):
         data = column.build_categorical_column(
             categories=categories, codes=codes, ordered=ordered
         )
-    elif is_string_dtype(dtype):
+    elif is_string_dtype(s.dtype):
         data = ["cat", "dog"]
     else:
-        data = np.arange(start=0, stop=2, dtype=dtype)
+        data = np.arange(start=0, stop=2, dtype=s.dtype)
```

Comment:
Any reason not to generate this data on the gpu using `cupy.arange`?

Reply:
It is because cupy doesn't support datetime, hence added a branch for numeric types to redirect a `cp.arange`

Reply:
IIRC we have `cudautils.arange` which dispatches to a `cupy.arange` and handles the datetime typecasting zero copy.

Reply:
`cudautils.arange` seems to be missing, are you referring to `as_contiguous `?: https://github.com/rapidsai/cudf/blob/branch-0.14/python/cudf/cudf/utils/cudautils.py#L22

Reply:
Nope, guess we removed it, but we could readd it and take the same approach to enable datetime aranges :)

Reply:
Had a discussion offline and decided to have a separate PR for adding back `cudautils.arange`, currently doing the operation in-line.